setup.cfg

```diff
@@ -0,0 +1,41 @@
+[metadata]
+name = postprocessing
+version = 2.5
+description = Post-processing agent to automatically catalog and reduce neutron data
+url = https://github.com/neutrons/post_processing_agent
+license = Open
```

Comment:
I am not sure what kind of license Open refers to.
If we are planning to publish this to PyPI, we should use BSD-3 or MIT, probably.